examples/proxy/psp22_upgradeable/lib.rs

```diff
@@ -0,0 +1,45 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![feature(min_specialization)]
+
+#[brush::contract]
+pub mod my_psp22_upgradeable {
+    use brush::contracts::psp22::*;
+    use ink_storage::traits::SpreadAllocate;
+    use ink_lang as ink;
+
+    #[ink(storage)]
+    #[derive(Default, SpreadAllocate, PSP22Storage)]
+    pub struct MyPSP22 {
```

Comment:
You also can use `Ownable` here and it will also show that owner of `Proxy` is owner of that code too